plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java

```diff
@@ -455,8 +455,8 @@ public void testShowCreateTable()
     @Override
     public void testReadMetadataWithRelationsConcurrentModifications()
     {
-        // TODO: Enable this test after fixing "Task did not completed before timeout"
-        throw new SkipException("TODO");
+        // TODO: Enable this test after fixing "Task did not completed before timeout" (https://github.com/trinodb/trino/issues/14230)
```

Comment:
https://github.com/trinodb/trino/issues/14230

Reply:
Is this comment just to create link between this PR and the issue? Or do I need to do something?

Reply:
> Is this comment just to create link between this PR and the issue? 

this